projects/packages/my-jetpack/global.d.ts

```diff
@@ -142,6 +142,7 @@ interface Window {
 					pricing_for_ui?: {
 						available: boolean;
 						wpcom_product_slug: string;
+						wpcom_free_product_slug: string;
```

Comment:
```suggestion
						wpcom_free_product_slug?: string;
```

This should be an optional property

Reply:
True thanks!